src/Sonarr.Api.V3/ProviderBulkResource.cs

```diff
@@ -9,6 +8,18 @@ public class ProviderBulkResource<T>
         public List<int> Ids { get; set; }
         public List<int> Tags { get; set; }
         public ApplyTags ApplyTags { get; set; }
+
+        public ProviderBulkResource()
+        {
+            Ids = new List<int>();
+        }
+    }
+
+    public enum ApplyTags
```

Comment:
I think moving this is the right idea, but it shouldn't be part of this resource, it can have it's own file instead.